organisation/src/main/java/org/cardanofoundation/lob/app/organisation/domain/entity/OrganisationChartOfAccountSubType.java

```diff
@@ -17,13 +16,14 @@
 @Getter
 @Setter
 @Entity
-@Table(name = "organisation_chart_of_account_sub_type")
+@Table(name = "organisation_chart_of_account_sub_type", indexes = {@Index(name = "astu_name", columnList = "name", unique = true)})
 @Audited
 @EntityListeners({AuditingEntityListener.class})
 public class OrganisationChartOfAccountSubType extends CommonEntity {
 
     @Id
-    private String id;
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
This is fine just be careful in the future with all the objects you want to put on the blockchain, it is better to use our own server side or client side ID generators which are deterministic and bound to business ids (natural id keys)